src/viadot/sources/__init__.py

```diff
@@ -68,3 +69,8 @@
     from viadot.sources.databricks import Databricks  # noqa: F401
 
     __all__.append("Databricks")
+
+if find_spec("pandas_gbq"):
+    from viadot.sources.bigquery import BigQuery
+
+    __all__.append("BigQuery")
```

Comment:
You haven't specified this as an optional dependency so the escape is not required.

Reply:
[6d00a97](https://github.com/dyvenia/viadot/pull/1035/commits/6d00a97a1f690365491fb1886e6b2fabcce79f2d)